site/docs/uk/guide/context.md

```diff
@@ -0,0 +1,575 @@
+---
+prev: ./basics.md
+next: ./api.md
+---
+
+# Контекст
+
+Об’єкт `Context` ([довідник API grammY](https://deno.land/x/grammy/mod.ts?s=Context)) є важливою частиною grammY.
+
+Кожного разу, коли ви реєструєте обробника на своєму об’єкті бота, цей обробник отримає об’єкт контексту.
+
+```ts
+bot.on("message", (ctx) => {
+  // `ctx` - це об'єкт `Context`.
+});
+```
+
+Ви можете використовувати об’єкт контексту, щоб
+
+- [отримати інформацію про повідомлення](#доступна-інформація),
+- [виконати дії у відповідь на повідомлення](#доступні-дії).
```

Comment:
This anchor is broken

Reply:
It's weird. In the chapter title, the last letter in the last word is "ї" (two dots on top), but the browser navigation box uses the letter "i" (one dot). If I use the letter "i" (one dot) in the anchor, it will be a warning in the code editor.

Reply:
```suggestion
- [виконати дії у відповідь на повідомлення](#%D0%B4%D0%BE%D1%81%D0%BB%D1%96%D0%B4%D0%B6%D0%B5%D0%BD%D0%BD%D1%8F-%D1%87%D0%B5%D1%80%D0%B5%D0%B7-has-%D0%BF%D0%B5%D1%80%D0%B5%D0%B2%D1%96%D1%80%D0%BA%D1%83).
```

Reply:
```suggestion
- [виконати дії у відповідь на повідомлення](#%D0%B4%D0%BE%D1%81%D1%82%D1%83%D0%BF%D0%BD%D1%96-%D0%B4%D1%96%D1%96).
```

Reply:
Escaping the characters would clearly be the reliable way, yes. But then again it may also be somewhat annoying that they aren't human-readable. You can try only escaping the problematic characters (in this case just ï?)